openshift/templates/dancer-mysql.json

```diff
@@ -329,87 +329,88 @@
   "parameters": [
     {
       "name": "MEMORY_LIMIT",
-      "displayName": "Memory limit",
+      "displayName": "Memory Limit",
       "description": "Maximum amount of memory the Perl Dancer container can use",
       "value": "512Mi"
     },
     {
       "name": "MEMORY_MYSQL_LIMIT",
-      "displayName": "Memory limit",
+      "displayName": "Memory Limit (MySQL)",
       "description": "Maximum amount of memory the MySQL container can use",
       "value": "512Mi"
     },
     {
       "name": "SOURCE_REPOSITORY_URL",
-      "displayName": "Source repository URL",
+      "displayName": "Source Repository URL",
```

Comment:
yeah i guess that is true.  "source repository" was aspirational but if we ever support others the template will have to change anyway.